### ISSUE TITLE
Investigate if we need to re-forward properties

### ISSUE THREAD

**Issue by nomego** [Neovici/cosmoz-data-nav#90]
When implementing "realign" for scenarios where items disappear from the item list, but other items that are rendered should stay intact (https://github.com/Neovici/cosmoz-data-nav/blob/82f234f8b55383be4dd1cf56cc7ac7eb5ed8bf34/cosmoz-data-nav.js#L598), we also re-forward the properties (https://github.com/Neovici/cosmoz-data-nav/blob/82f234f8b55383be4dd1cf56cc7ac7eb5ed8bf34/cosmoz-data-nav.js#L619 + 620).

This causes us to actually re-stamp/re-eval the same instance, which in turn, in some cases, causes the views to re-request information from BE unnecessarily.

Can we safely drop this property forwarding for existing, rendered elements or why is it there in the first place?

This can be tested in Cosmoz FE view "Purchase -> Orders -> Orders rows with quantity differences" -> Queue -> action "Approve order row".
You will notice that history and suggestions are called twice, even though we only request information for one new row.

**Comment by cristinecula** [comment]
@nomego I tried disabling the prop forwarding, by removing those lines in _realignElements, but (aside from causing errors in the tests) the multiple requests for the same id appear to continue to be happening.
The data-nav logic is very complex and hard to follow, so more investigation is needed. 
In the long run, I think separating the data layer from the view layer would result in more benefits, than patching this library. The view layer should not make API requests directly, but instead only display the data. If we do this correctly, then we can forward props all the time, without triggering network requests.
